presto-main/src/main/java/com/facebook/presto/sql/planner/optimizations/UnaliasSymbolReferences.java

```diff
@@ -595,11 +596,10 @@ private void map(Symbol symbol, Symbol canonical)
             mapping.put(symbol, canonical);
         }
 
-        private Assignments canonicalize(Assignments oldAssignments)
+        private void map(Assignments assignments)
```

Comment:
The verb "map" makes me thing it will create something and return it.  How about a name suggestive of side-effects, like "addMappings" or "gatherAliases"?